tests/integration_tests/file_interface_serpent/test.py

```diff
@@ -36,6 +36,54 @@ def test_runtime_input_from_template(serpent_depcode, msr):
     file_data = serpent_depcode.create_runtime_matfile(file_data)
     assert file_data[0].split()[-1] == '\"' + \
         serpent_depcode.runtime_matfile + '\"'
+
+    # get_burnable_material_card_data
+    _burnable_material_card_data = {'fuel':
```

Comment:
Maybe this is just a knowledge gap on my part, but why did you add the underscore here? This looks like a variable to me, not a call of that private function?

Reply:
I get that for in line 79. But even when you define it here and use it in line 80? It's still a private attribute in those locations?

Reply:
no, this is just for naming consistency. Should I name it something else?

Reply:
I would recommend changing the same some when you're not calling the attribute. It was a little confusing to me to have the two things you're comparing be named the same thing. I like to use the `exp` and `obs` (as in expected and observed) that Katy uses in her book for my tests. But your choice as to what you change it to.

Reply:
Okay I'll do this